pkg/server/handler/v1alpha1/integration.go

```diff
@@ -108,12 +126,31 @@ func createIntegration(tenant string, in *api.Integration) (*api.Integration, er
 }
 
 // GetIntegration gets an integration with the given name under given tenant.
-func GetIntegration(ctx context.Context, tenant, name string) (*api.Integration, error) {
-	return getIntegration(tenant, name)
+func GetIntegration(ctx context.Context, tenant, name string, includePublic bool) (*api.Integration, error) {
+	integration, err := getIntegration(svrcommon.TenantNamespace(tenant), name)
```

Comment:
Use `isPublic` instead  `includePublic` for name. And check whether it's public integration first